gradle.properties

```diff
@@ -20,11 +20,11 @@ buildKotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds
 remoteRoot=konan_tests
 kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.40-dev-2423,branch:default:true,pinned:true/artifacts/content/maven
 kotlinVersion=1.3.40-dev-2423
-testKotlinVersion=1.3.40-dev-2423
+testKotlinVersion=1.3.40-dev-2454
```

Comment:
Is it properly pinned?

Reply:
@olonho This is a work-in-progress pull request, so please disregard `gradle.properties` changes.
@qurbonzoda Better to mark this PR as draft until it's ready to avoid confusion.

Reply:
Replaced `WIP` with `Draft` in PR title, but unfortunately marking an existing PR as draft is not possible yet: https://github.community/t5/How-to-use-Git-and-GitHub/Feature-Request-Switch-from-ready-to-draft-in-pull-requests/td-p/19107